tests/repos/test_cli.py

```diff
@@ -75,6 +75,18 @@ def test_create_validation(repos_api_mock):
     )
 
 
+@provide_conf
+def test_create_validation_partial(repos_api_mock):
+    runner = CliRunner()
+    runner.invoke(cli.create_repo_cli,
+                  ["--url", TEST_URL])
+    repos_api_mock.create.assert_called_once_with(
```

Comment:
can you also add a test for `awsCodeCommit`, as it has the deepest logic?..

Reply:
Added tests...